stripe/src/main/java/com/stripe/android/view/CardInputView.java

```diff
@@ -26,7 +26,7 @@
     private static final int END_INDEX_AMEX = 11;
 
     private CardNumberEditText mCardNumberEditText;
-    private EditText mCvcNumberEditText;
+    private DeleteWatchEditText mCvcNumberEditText;
```

Comment:
Could we, sometime in the future, create a `StripeEditText` that could support basic callbacks like `onDeleteCallback`, `onCompleteCallback` and have subclasses declare the `delete` and `complete` criteria?